internal/venice-client-common/src/main/java/com/linkedin/venice/stats/metrics/MetricEntityState.java

```diff
@@ -102,16 +111,65 @@ void recordTehutiMetric(double value) {
     }
   }
 
-  public void record(long value, Attributes otelDimensions) {
-    recordOtelMetric(value, otelDimensions);
+  protected final void record(long value, Attributes attributes) {
+    recordOtelMetric(value, attributes);
     recordTehutiMetric(value);
   }
 
-  public void record(double value, Attributes otelDimensions) {
-    recordOtelMetric(value, otelDimensions);
+  protected final void record(double value, Attributes attributes) {
```

Comment:
Nit, can we make these two generic? We have such separate calls in almost every subclass of MetricEntityState. It would be great if we can generalize them and save many similar codes.
```
record(long, Attributes attributes)
record(double, Attributes attributes)
```

Reply:
No, not really. We can go with it for now.